docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -350,6 +350,23 @@ The following table includes some additional commands that might help you manage
 | Runs one iteration of `logrotate` manually, without changing the scheduled interval. | `sudo logrotate -f /etc/logrotate.conf --force` |
 | Reloads `syslog-ng`. | `sudo /usr/sbin/syslog-ng-ctl reload` |
 
+### UID2 Operator error codes
+
+The following table lists possible errors while setting up private operator.
```

Comment:
Suggest:
The following table lists some errors you might encounter when setting up a Private Operator.

(generally we initial cap Private Operator in the docs)